src/node.js

```diff
@@ -77,7 +77,11 @@ const Node = AsyncInit.compose({
     if (!url) throw new Error('"url" required')
     this.url = url.replace(/\/$/, '')
     this.internalUrl = internalUrl ? internalUrl.replace(/\/$/, '') : this.url
-    const client = await genSwaggerClient(`${this.url}/api`, { internalUrl: this.internalUrl })
+    let client = await genSwaggerClient(`${this.url}/api`, { internalUrl: this.internalUrl })
+    if (client.spec.info.version.split('-')[0] === '0.0.0') {
```

Comment:
What shall we do with this `0.0.0`? Iris is planned as `6.0.0` 🤔

Reply:
I would replace it with something meaningful before the release, currently, it works with that we have 🙃

Reply:
Actually, I have something to propose, let's increase the upper bound of the node version to < 7.0.0 (implemented in the last commit). Would be nice to merge it because after that we will be able to safely refer to this branch as a part of develop branch.